lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -280,10 +280,10 @@ def call(env)
             env[:ui].info(" -- smartcard device:  mode=#{@smartcard_dev[:mode]}, type=#{@smartcard_dev[:type]}")
           end
 
-          @qargs = config.qemu_args
-          if not @qargs.empty?
+          @qemu_args = config.qemu_args
```

Comment:
If you see other variables on create_domain.rb, there are a lot of variables that just use the same name as config.rb.
If we want to use different name of variables, we need to make both create_domain.rb and domain_spec.rb use same variables between config.rb and create_domain.rb since create_domain.rb redefines variables itself before generating domain while domain_spec.rb refers variables from config.rb.